srtcore/buffer.cpp

```diff
@@ -924,7 +922,10 @@ void CRcvBuffer::skipData(int len)
       m_iMaxPos = 0;
 }
 
-bool CRcvBuffer::getRcvFirstMsg(ref_t<uint64_t> r_tsbpdtime, ref_t<bool> r_passack, ref_t<int32_t> r_skipseqno, ref_t<int32_t> r_curpktseq)
```

Comment:
Until the `ref_t` is finally removed.

Reply:
Both are wrong. Changes should concern exactly one topic. The PR for removal of `ref_t` is underway and it will be done at some point once we deal with previous things. Here should be simply `ref_t` restored, with the `r_` prefix as well. The change for the clock should only replace `uint64_t` with `steady_clock::time_point`. All other changes should be restored.

This change, even if thought as a `ref_t` removal, is done completely wrong way.

Reply:
`r_` prefix would be wrong according to the new convention, true. Depends on whether you want to remain with old convention, or apply the new one.

Reply:
Applied new style.